### DISCUSSION TITLE
Does this plugin still work in 2024?

### DISCUSSION THREAD

**Discussion by ijrc01** [homebridge-simplisafe3/homebridge-simplisafe3#445] [Q&A] [answered]
### Description

I cant for the life of me figure out how to auth. I get this error page after signing in, but dont know where to find this hidden url that the docs reference

### Expected behavior

eazy auth

### Steps to reproduce

click purple button, open link, sign in, and somehow magically find the URL that begins with com.SimpliSafe.mobile://

### Homebridge logs

```bash
na
```

### Homebridge config.json contents

```json
na
```

### Screenshots

![image](https://github.com/user-attachments/assets/7f52ab0e-6cae-4408-8d21-80c08d0b2879)

### Plugin version

1.10.13

### Homebridge version

1.8.5

### Node.js version

18.20.5

### Host OS

Docker

**Comment by ijrc01** [comment]
more screenshots w/ dev console open 
![image](https://github.com/user-attachments/assets/4792382c-9220-4402-bdc4-1524d1ab91a0)

**Comment by ijrc01** [comment]
![image](https://github.com/user-attachments/assets/d910e999-c7ab-4315-be66-f1c3c84aa1a9)

**Comment by shamoon** [comment, marked as answer]
> ### Expected behavior
> eazy auth

I don’t think this is actually a bug report.

I have no idea what those screenshots are but if you can’t login to the website then you won’t be able to setup auth. Certainly the screenshot implies you’ve disabled cookies somehow.

And yes, I use it every single day and I’m not aware of any issues like this.

**Reply by ijrc01** [reply]
I think this is a bug in the auth process of this plugin

**Reply by ijrc01** [reply]
I've tried in multiple browser on the same pc, and on both a mac and windows machine. same outcome.

**Comment by ijrc01** [comment]
Got it. Guys - you need better docs. You're making this unnecessarily tough for people. 

![image](https://github.com/user-attachments/assets/88314fca-7ba0-4327-83b4-cb50ea533ad2)
